bin/index.js

```diff
@@ -177,7 +187,18 @@ var debugLog = (log) => {
 
     }
 
-    const server = http.createServer((request, response) => {
+    if (options.ssl) {
+        securityConfig = {
```

Comment:
if options.ssl is true user most provide key and cert too, otherwise we have to show a proper message and exit.